openrpc.yaml

```
@@ -719,7 +721,10 @@ methods:
     tags:
       - $ref: '#/components/tags/MetaMask'
       - $ref: '#/components/tags/Restricted'
-    description: Returns a list of addresses for the accounts owned by the user.
+    description: >-
+      Returns a list of addresses for the accounts owned by the user.        
+      This method requires calling `wallet_requestPermissions` for permission. 
+      Alternatively, you can use [`eth_requestAccounts`](/wallet/reference/eth_requestAccounts). 
```

```suggestion
      This method requires calling `wallet_requestPermissions` for permission. 
      We recommend using `eth_requestAccounts`, which internally calls `wallet_requestPermission`.
```

Right now, we're not using links in the api-specs method descriptions to minimize overhead. So we can remove the link here for consistency.